cypress/support/component.tsx

```diff
@@ -63,8 +64,8 @@ declare global {
   }
 }
 
-Cypress.Commands.add('mount', (component, route) => {
-  cy.fixture('activeUser').then((activeUser: User) => {
+Cypress.Commands.add('mount', (component, route, fixture) => {
```

Comment:
fixture is too generic here. it's the fixture for the active user correct?
maybe activeUserFixture?